### PR TITLE
Add a bowl of rubber

### LINKED ISSUE #8: Rubber system
Normally industry mods should generate rubber tree, which is source of rubber.

**Describe the solution you'd like**

- [x] Generate rubber tree
- [ ] Can get rubber from the tree

### PR DESCRIPTION
**Pull request by longfangsong**
Close #8.

### CODE REVIEW

File: src/main/resources/assets/magic_crystal/models/block/rubber_log_producing.json [hunk 1]
```diff
@@ -0,0 +1,1364 @@
+{
+	"credit": "Made with Blockbench",
```
Comment (longfangsong) [inline src/main/resources/assets/magic_crystal/models/block/rubber_log_producing.json:2]: Download one if necessary!

File: src/main/resources/assets/magic_crystal/lang/zh_cn.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "item.magic_crystal.hammer": "锤子",
+  "item.magic_crystal.raw_tin": "粗锡",
+  "item.magic_crystal.iron_plate": "铁板",
+  "item.magic_crystal.rubber_bowl": "一碗橡胶",
```
Comment (zhoudian64) [inline src/main/resources/assets/magic_crystal/lang/zh_cn.json:5]: Should we separate rubber and latex?

File: src/main/kotlin/com/fairchild_superconductor/magic_crystal/rubber_tree/RubberLogEntity.kt [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    }
+
+    override fun writeNbt(nbt: NbtCompound?): NbtCompound {
+        nbt?.putBoolean("HasBowl", !isEmpty)
+        nbt?.putInt("TicksPassed", ticksPassed)
+        return super.writeNbt(nbt)
+    }
+
+    override fun tick(world: World?, pos: BlockPos?, state: BlockState?, blockEntity: RubberLogEntity?) {
+        if (!containedItem.isEmpty && ticksPassed < 100) {
```
Comment (zhoudian64) [inline src/main/kotlin/com/fairchild_superconductor/magic_crystal/rubber_tree/RubberLogEntity.kt:82]: Strange design...

**Review by zhoudian64 — APPROVED** [review]
rest looks good to me

File: src/main/kotlin/com/fairchild_superconductor/magic_crystal/rubber_tree/RubberLogEntity.kt (hunk 3 above)
Comment (longfangsong) [reply src/main/kotlin/com/fairchild_superconductor/magic_crystal/rubber_tree/RubberLogEntity.kt:82]: Do you have any suggestions?
I really don't know what's the expected behaivour here.

File: src/main/resources/assets/magic_crystal/lang/zh_cn.json (hunk 2 above)
Comment (longfangsong) [reply src/main/resources/assets/magic_crystal/lang/zh_cn.json:5]: There will be a "synthetic rubber" item.

File: src/main/kotlin/com/fairchild_superconductor/magic_crystal/rubber_tree/RubberLogEntity.kt (hunk 3 above)
Comment (zhoudian64) [reply src/main/kotlin/com/fairchild_superconductor/magic_crystal/rubber_tree/RubberLogEntity.kt:82]: 100 ticks look really laggy while playing with a high performance computer.

Comment (longfangsong) [reply src/main/kotlin/com/fairchild_superconductor/magic_crystal/rubber_tree/RubberLogEntity.kt:82]: > 100 ticks look really laggy while playing with a high performance computer.

This is meant to be slow —— rubber takes time to produce.
100 ticks is so so quick when compare to real world.
However, maybe we should make this process clear to the player, for example, we can:
- Add an UI to show the process
- Use another model with a bowl "inside" the log to show this log is producing rubber.
I'm just not sure which one is better...

**Pull request merged**